### PR TITLE
Backport listing template from dlr

### PR DESCRIPTION
**Pull request by iRohitSingh**

### CODE REVIEW

**Comment by sneridagh** [comment]
@iRohitSingh CI is red, can you fix it?

**Comment by iRohitSingh** [comment]
@sneridagh this pr depends on volto-blocks. We have to configure it using the mrs.developer. Once the pr of yours merged I will start working on this.

**Comment by iRohitSingh** [comment]
@sneridagh everything is ready

File: src/customizations/volto/components/manage/Blocks/Listing/ListingBody.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    ListingBodyTemplate = legacyTemplateConfig.template;
+  } else {
+    ListingBodyTemplate =
+      variation?.template ?? defaultVariation?.template ?? null;
+  }
+  let galleryRef;
+
+  if (data.variation === 'imageGallery') {
+    galleryRef = useRef();
+  }
```
[the remark this answers is not in the document]
Comment (iRohitSingh) [reply src/customizations/volto/components/manage/Blocks/Listing/ListingBody.jsx:43]: done

**Comment by sneridagh** [comment]
@iRohitSingh please check CI.

**Comment by iRohitSingh** [comment]
Done

**Pull request merged**